src/main.c

```diff
@@ -328,7 +328,13 @@ static uint32_t softdev_init(bool init_softdevice)
       .accuracy     = NRF_CLOCK_LF_ACCURACY_250_PPM
   };
 
+#ifdef BLE
   APP_ERROR_CHECK( sd_softdevice_enable(&clock_cfg, app_error_fault_handler) );
+#elif defined(BOTH)
+  APP_ERROR_CHECK( sd_softdevice_enable(&clock_cfg, app_error_fault_handler, ANT_LICENSE_KEY ) );
```

Comment:
I didn't see where `ANT_LICENSE_KEY` is defined anywhere. Are you putting it in the `board.h` ?

Reply:
ANT_LICENSE_KEY is defined in nrf_sdm.h for the dualstack softdevices. It is commented out by default as a notice that it is only to be used for evaluation purposes. Therefore, I don't think it would be legal for myself to define this.

When a user downloads the correct softdevice headers and compiles against them, this will be defined.

Reply:
thanks, this is not too obvious to me